schema/one-time-polled-screening.json

```diff
@@ -861,7 +861,7 @@ Note that, as this is a demo check, no credits will actually be deducted from th
                         "items": {
                             "$ref": "#/components/schemas/Charge"
                         },
-                        "description": "A list of charges incurred while running this check. Charges may only be returned if no errors occurred (i.e. a result was generated) and the check is being resold (the commercial relationship is with PassFort). For demo checks, charges may be returned, but they will not be deducted from the institution's credit. If charges are returned, the total amount charged must be no greater than the maximum cost-per-check specified in the integration's configuration."
+                        "description": "A list of charges incurred while running this check. Charges may only be returned if no errors occurred (i.e. a result was generated) and the check is being resold (the commercial relationship is with Passfort). For demo checks, charges may be returned, but they will not be deducted from the institution's credit. If charges are returned, the total amount charged must be no greater than the maximum cost-per-check specified in the integration's configuration."
```

Comment:
Remove brackets and suggest shortening:

"Charges may only be returned if there are no errors when the check is run and the check is being resold by Passfort."